src/guiapi/include/screen_unloop.h

```diff
@@ -0,0 +1,19 @@
+//screen.h
+#pragma once
+#include "stdint.h"
+#include "screen.h" //screen.t
+
+#ifdef __cplusplus
+extern "C" {
+#endif //__cplusplus
+
+typedef enum {
+    sculp_timeout,
+    sculp_M876
```

Comment:
It took me some time to understand that `sculp` is short for `screen unlooping` (and in this case, I had the full context). In case I would find this at a random place in the codebase, I am quite sure I would have no idea what it means.

Can we rename it? `SCREEN_UNLOOPING_TIMEOUT`, `SCREEN_UNLOOPING_M876`...


Hmm, but what is the semantics of „screen unlooping“? From what I see, it is about „closing as many screens as possible“ – but there are _no loops involved_. 

What about just `close_screens(CLOSE_ON_M876)`?

Reply:
screen_close() also opens previous screen
i think unlooping is fine

Reply:
sculp_timeout, sculp_M876
renamed